### DISCUSSION TITLE
Extract Category when available

### DISCUSSION THREAD

**Discussion by ryflamm** [mealie-recipes/mealie#4779] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Many online recipes have categories defined in the recipe schema. Currently Mealie does not import these values.

### Please provide a concise description of the feature that would resolve your issue.

On import page, add a checkbox to import categories (similar to 'keywords as tags')

### Please consider and list out some caveats or tradeoffs made in your design decision

Feature may not be universally desired. Making optional checkbox resolves this.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Gtt1229** [comment]
I've created https://github.com/mealie-recipes/mealie/pull/6523 for this.
